### PR TITLE
update suite2p args

### PR DESCRIPTION
**Pull request by bendichter**
The extra args in roiextractor say they should be deprecated by now, so let's do that here first.

https://github.com/catalystneuro/roiextractors/blob/22ccdad19697284efa5b39db09d72e9d598eac81/src/roiextractors/extractors/suite2p/suite2psegmentationextractor.py#L97-L111

Also I added some notes because I found interacting with this interface in GUIDE a bit confusing. I wasn't sure if I was supposed to select a the plane# directory. I also was unsure whether every plane would be loaded. I think we should build a converter that loads all of the frames, but that can be in another PR

### CODE REVIEW

File: src/neuroconv/datainterfaces/ophys/suite2p/suite2pdatainterface.py [hunk 1]
```diff
@@ -47,6 +47,18 @@ class Suite2pSegmentationInterface(BaseSegmentationExtractorInterface):
...
 
+    @classmethod
+    def get_source_schema(cls) -> dict:
+        schema = super().get_source_schema()
+        schema["properties"]["folder_path"][
+            "description"
+        ] = "Path to the folder containing Suite2p segmentation data. Should contain 'plane#' subfolder(s)."
+        schema["properties"]["plane_name"][
+            "description"
+        ] = "The name of the plane to load. This interface only loads one plane at a time. Use the full name, e.g. 'plane0'. If this values is omitted, the first plane found will be loaded."
```
Comment (CodyCBakerPhD) [inline src/neuroconv/datainterfaces/ophys/suite2p/suite2pdatainterface.py:58]: Was it missing the argument entirely or just a lackluster description?

Comment (bendichter) [reply src/neuroconv/datainterfaces/ophys/suite2p/suite2pdatainterface.py:58]: It didn't have a description at all

**Comment by bendichter** [comment]
In this screen

image


it isn't clear to me whether I should be selecting "plane0" or the directory that contains it

File: src/neuroconv/datainterfaces/ophys/suite2p/suite2pdatainterface.py [hunk 2]
```diff
@@ -66,18 +78,20 @@ def __init__(
...
         """
 
         Parameters
         ----------
         folder_path : FolderPathType
+            Path to the folder containing Suite2p segmentation data. Should contain 'plane#' sub-folders.
         channel_name: str, optional
-            The name of the channel to load, to determine what channels are available use Suite2pSegmentationInterface.get_available_channels(folder_path).
+            The name of the channel to load.
+            To determine what channels are available, use ``.get_available_channels(folder_path)``.
```
Comment (bendichter) [inline src/neuroconv/datainterfaces/ophys/suite2p/suite2pdatainterface.py:90]:
```suggestion
            To determine what channels are available, use ``Suite2pSegmentationInterface.get_available_channels(folder_path)``.
```

File: src/neuroconv/datainterfaces/ophys/suite2p/suite2pdatainterface.py [hunk 3]
```diff
@@ -66,18 +78,20 @@ def __init__(
...
+            Path to the folder containing Suite2p segmentation data. Should contain 'plane#' sub-folders.
         channel_name: str, optional
-            The name of the channel to load, to determine what channels are available use Suite2pSegmentationInterface.get_available_channels(folder_path).
+            The name of the channel to load.
+            To determine what channels are available, use ``Suite2pSegmentationInterface.get_available_channels(folder_path)``.
         plane_name: str, optional
-            The name of the plane to load, to determine what planes are available use Suite2pSegmentationInterface.get_available_planes(folder_path).
+            The name of the plane to load. This interface only loads one plane at a time.
+            If this value is omitted, the first plane found will be loaded.
+            To determine what planes are available, use ``.get_available_planes(folder_path)``.
```
Comment (bendichter) [inline src/neuroconv/datainterfaces/ophys/suite2p/suite2pdatainterface.py:94]:
```suggestion
            To determine what planes are available, use ``Suite2pSegmentationInterface.get_available_planes(folder_path)``.
```

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**